sdk-cpp/tests/test_data.cpp

```diff
@@ -10,11 +10,19 @@ const uint64_t g_smallFileSizeBytes = 1837u;
 const uint64_t g_largeFileSizeBytes = 536870440u;
 const uint64_t g_prodFileSizeBytes = 25006511u;
 
+std::string downloadPath = "/var/lib/deliveryoptimization-snap-downloads-root";
+
+// #ifdef DO_BUILD_FOR_SNAP
```

Comment:
For the ifdef to work, you have to pass DO_BUILD_FOR_SNAP into the sdk tests compilation via sdk-cpp/tests/CMakeLists.txt. See how the DO agent cmakelists does it.